### ISSUE TITLE
Incorrect Armed State

### ISSUE THREAD

**Issue by WorldOfMaze** [excaliburpartners/HAILogger#40]
In think I have discovered an issue with OmniLinkBridge when arming to vacation mode.  I am running the most current version of Home Assistant using OmniLinkBridge with MQTT.

Here is a snippet of the history.

```
[MAIN] changed to Armed away
9:03:13 AM - 9 hours ago

[MAIN] changed to Arming by service alarm_control_panel.alarm_arm_vacation
9:02:13 AM - 9 hours ago

[MAIN] changed to Disarmed
7:13:25 AM - 10 hours ago
```

Note that the state was set _Arming_ rather than _Vacation_.  My OmniPro has different automations for _Vacation_ mode which, among other things, lowers the temperatures on my t-stats so this is important.

Not sure if this is a known issue or if there is a work around.

Please let me know if there is any further information you need.

Thank you for making this integration available.  Without it, I would not have a working smart home.

**Comment by rwagoner** [comment]
The state will change to arming for the exit delay before going to the mode (home, away, night). At the time Home Assistant didn't support vacation mode. OmniLinkBridge mapped vacation to away in the area/basic_state topic. The area/state topic does show armed_vacation. However looking at the Home Assistant code I see they added support for vacation mode last August. I will test and change the mapping for the next release.

**Comment by WorldOfMaze** [comment]
Awesome!  Thank you, @rwagoner

**Comment by WorldOfMaze** [comment]
@rwagoner I am currently running OmniLinkBridge 1.1.11.0 on Home Assistant.  Do you have any thoughts on when you will be able to publish tne next release with  Vacation Mode support?  I love the OmniLinkBridge and, for me, this one fix would make it perfect.

Thank you, Ryan.

**Comment by rwagoner** [comment]
I've been testing the vacation mode changes in the dev branch for a few weeks. Everything looks good so I just released 1.1.12. Enjoy!

**Comment by WorldOfMaze** [comment]
Thank you, @rwagoner.   I am installing 1.1.12.
